examples/autonat/src/bin/autonat_client.rs

```diff
@@ -49,20 +49,17 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let opt = Opt::parse();
 
-    let local_key = identity::Keypair::generate_ed25519();
-    let local_peer_id = PeerId::from(local_key.public());
-    println!("Local peer id: {local_peer_id:?}");
+    let mut swarm = libp2p::builder::SwarmBuilder::new()
```

Comment:
```suggestion
    let mut swarm = libp2p::SwarmBuilder::new()
```

What do you think of re-exporting this?

Reply:
:+1: https://github.com/libp2p/rust-libp2p/pull/4120/commits/cb67678abdb92be7f9f68dbff66968224820a25c